deploy/helm/charts/Chart.yaml

```diff
@@ -16,4 +16,5 @@ sources:
 - https://github.com/openebs/zfs-localpv
 dependencies:
   - name: crds
-    version: "2.5.0"
+    version: "2.5.1"
```

Comment:
nit: nit: this shouldn't have the double quotes because we're typically not using quotes in chart versions.